app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -45,21 +45,30 @@ def configure_imported_vm_networks(vm_id)
     end
   end
 
+  def check_task!(t, msg)
+    raise _(msg) if t.nil? || MiqTask.status_error?(t.status) || MiqTask.status_timeout?(t.status)
+  end
+
   def submit_import_vm(userid, source_vm_id, target_params)
     task_id = queue_self_method_call(userid, 'Import VM', 'import_vm', source_vm_id, target_params)
     task = MiqTask.wait_for_taskid(task_id)
 
+    check_task!(task, 'Error while importing the VM.')
+
     task.task_results
   end
 
   def validate_import_vm
-    Gem::Version.new(api_version) >= Gem::Version.new('4')
+    # The version of the RHV needs to be at least 4.1.5 due to https://bugzilla.redhat.com/1477375
+    !api_version.nil? && Gem::Version.new(api_version.split('_').first) >= Gem::Version.new('4.1.5')
```

Comment:
Could it be that `api_version.nil` have a nill value ?

Please `use version_higher_than?` from infra_manager for comparing versions.